services/maven-metadata/maven-metadata.tester.js

```diff
@@ -12,6 +12,33 @@ t.create('valid maven-metadata.xml uri')
     message: isVPlusDottedVersionAtLeastOne,
   })
 
+t.create('version ending with zero')
+  .get(
+    '/v.json?metadataUrl=http://central.maven.org/maven2/mocked-group-id/mocked-artifact-id/maven-metadata.xml'
+  )
+  .intercept(nock =>
+    nock('http://central.maven.org/maven2')
+      .get('/mocked-group-id/mocked-artifact-id/maven-metadata.xml')
+      .reply(
+        200,
+        `
+      <metadata>
+        <groupId>mocked-group-id</groupId>
+        <artifactId>mocked-artifact-id</artifactId>
+        <versioning>
+          <latest>1.30</latest>
+          <release>1.30</release>
+          <versions>
+            <version>1.30</version>
+          </versions>
+          <lastUpdated>20190902002617</lastUpdated>
+        </versioning>
+      </metadata>
+      `
+      )
+  )
+  .expectBadge({ label: 'maven', message: /^v1\.30$/ })
```

Comment:
Any reason the regex is needed for `message`? Would the below not work?

```suggestion
  .expectBadge({ label: 'maven', message: 'v1.30' })
```

Reply:
Not at all, apart from keeping the same convention from the source this snippet is adapted from. It should work without the regex, indeed. Shall we remove it then?

(And BTW should we leave the [same line](https://github.com/badges/shields/blob/54bcedc0f42a211e8b802001e28af27da024a621/services/maven-central/maven-central.tester.js#L53) at `maven-central` as-is in this PR? If you like this PR to change that line for `maven-central.test.js`, we can include it as well.)

Reply:
Sure that sounds good

Reply:
Updated via a69a796 and da735da.